example/app.py

```diff
@@ -36,10 +41,12 @@
 
 routes = [
     Route("/", homepage),
-    WebSocketRoute("/", chatroom_ws, name='chatroom_ws'),
+    WebSocketRoute("/", chatroom_ws, name="chatroom_ws"),
 ]
 
 
 app = Starlette(
     routes=routes, on_startup=[broadcast.connect], on_shutdown=[broadcast.disconnect],
 )
+
+uvicorn.run(app, log_level="info", loop="asyncio")
```

Comment:
I'm not sure how keen I am on running `uvicorn` programmatically.

(Eg. better support for running multiple instances, reloading, etc. if running it as a cli, which I tend to push for in most places.)